cukinia

```diff
@@ -74,7 +142,20 @@ logging()
 # cukinia_log: log arguments to stdout
 cukinia_log()
 {
-	echo "${__log_pfx}$*"
+	local message=$1
+	local result=$2
+
+	case ${__log_format} in
+	csv)
+		if [ -n "$result" ]; then
+			# Escape single quotes in test message to avoid commas detection (line splitting avoided using single quotes)
```

Comment:
style: fixup line so that it fits in 80col. Bonus: rephrase to be a short 50char comment